src/Stripe.net/Entities/SubscriptionSchedules/SubscriptionSchedulePhaseItem.cs

```diff
@@ -6,18 +6,32 @@ namespace Stripe
 
     public class SubscriptionSchedulePhaseItem : StripeEntity<SubscriptionSchedulePhaseItem>
     {
+        /// <summary>
+        /// Define thresholds at which an invoice will be sent, and the related subscription
+        /// advanced to a new billing period.
+        /// </summary>
         [JsonProperty("billing_thresholds")]
-        public SubscriptionItemBillingThresholds BillingThresholds { get; set; }
+        public SubscriptionSchedulePhaseItemBillingThresholds BillingThresholds { get; set; }
```

Comment:
Is this one normal?

Reply:
yeah I think this is directly related to removing those overrides.